DIRECTOR_HANDBOOK.md

```diff
@@ -0,0 +1,92 @@
+
+It is the express policy of the Foundation to require that all its activities be conducted strictly in accordance with federal and state laws. It is extremely important that all members of the Board of Directors be aware of the legal and ethical standards to which you are held and the types of activities to avoid in order to fulfill your duties as directors properly, as well as to avoid liability.
+
+These Governance Guidelines, Code of Conduct and Duties were prepared to provide you with a broad, summary overview of the standards under the law to which you will be held. However, you should note that these governance guidelines and standards provide a general guide only; they are not intended to be a complete and definitive statement of all aspects of the law relating to the duties of Directors. Any specific questions relating to your duties as a Director not addressed in these governance guidelines and standards should be referred to counsel for the Foundation or to your own legal counsel who has responsibility for considering your duties as a Director of the Foundation.
+
+## Board Governance
+
+The Board of Directors represents the interests of the Foundation’s Members. The Board is the ultimate decision-making body of the Foundation except with respect to those matters reserved to the Members. The business and affairs of the Foundation are managed by its Board. Although the Board may delegate authority, the ultimate responsibility for the Foundation resides with the Board, which may not lawfully abdicate its duties.
+
+Directors need to be aware that the Foundation’s funds are dedicated to the purposes set forth in the Certificate of Incorporation, and may only be used in a manner consistent with the Foundation’s authorized tax-exempt purposes.
+
+Generally, the Board may take action at meetings at which the requisite number of Directors is participating (in person or via communications equipment), and the requisite number Directors vote in favor, in each case as provided in Article IV of the By-Laws. Actions may also be taken electronically in accordance with the rules of the same Article. In some cases, a Super Majority (two-thirds) of the then serving Directors is required to approve an action.
+
+Additional information about Board governance and Directors is contained in Article IV of the By-Laws, which all Directors should review carefully.
+
+## Core Responsibilities
+
+The Board’s core responsibilities consist of the following:
+
+* Advising and counseling management regarding significant issues facing the Foundation;
```

Comment:
We contract the Linux Foundation as our management company. There are various resources there that work on the Foundation's behalf.